src/Arcus.Observability.Tests.Unit/Telemetry/Logging/HttpDependencyLoggingTests.cs

```diff
@@ -426,6 +426,38 @@ public void LogHttpDependencyWithRequestWithDurationMeasurementWithDependencyId_
             Assert.ThrowsAny<ArgumentException>(() => logger.LogHttpDependency(request, statusCode, measurement: null, dependencyId));
         }
 
+        [Theory]
+        [InlineData(99)]
+        [InlineData(600)]
```

Comment:
To be even more secure (and without the lovely property-based testing), we should be more general in generating inputs here.

This will give us a bit more security:
```csharp
int statusCode = BogusGenerator.PickRandom(
    BogusGenerator.Random.Int(max: 99), 
    BogusGenerator.Random.Int(min: 600));
```

Reply:
done